.buildkite/premerge.steps.yaml

```diff
@@ -43,6 +44,7 @@ linux: &linux
     - "platform=linux"  # if you need a different platform, configure this: macos|linux|windows.
     - "queue=v4-20-07-06-120608-bk13080-eb89af6c"
     - "scaler_version=2"
+    - "maximum_instances=12"
```

Comment:
We don't need this on Linux as there are no Unity installs/licenses on the Linux agents :)